### PR TITLE
CASSANDRA-16488 added docs for Plausible deployment

### PR DESCRIPTION
**Pull request by smiklosovic**

### CODE REVIEW

File: plausible/README.adoc [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+----
+
+The main configuration file is `plausible-conf.env`
+
+----
+ADMIN_USER_EMAIL=stefan.miklosovic@instaclustr.com
+ADMIN_USER_NAME=admin
+ADMIN_USER_PWD=you will log in with this password under ADMIN_USER_EMAIL
+BASE_URL=https://plausible.cassandra.apache.org/
+SECRET_KEY_BASE=fMHBLyITacfbah4DF7qvnPydAkiE6h1cbNRbv0jDZzJ3q61uNyZ173Epx9pPVPkn
```
Comment (michaelsembwever) [inline plausible/README.adoc]: is this ok to be public? and if not, I think we need to now rotate it…

File: plausible/README.adoc [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+the other Docker Compose deployment.
+
+Plausible service is hosted under https://www.instaclustr.com[Instaclustr]'s account. Please contact
+stefan dot miklosovic at instaclustr dot com for any related questions / issues.
+
+Domain `plausible.cassandra.apache.org` points to 3.124.255.5. Please contact gmcdonald at apache dot org for
+any related questions / issues.
+
+For the access to all credentials (PEM to ssh to that box, logins to Plausible etc), please contact
+mck at apache dot org.
```
Comment (michaelsembwever) [inline plausible/README.adoc]:
```suggestion
For the access to all credentials (PEM to ssh to that box, logins to Plausible etc), please contact
someone on the PMC, as credentials are stored in the private subversion repository.
```

File: plausible/README.adoc (hunk 1 above)
Comment (smiklosovic) [reply plausible/README.adoc]: I rotated it and changed docs a bit.

**Review by michaelsembwever — APPROVED** [review]
LGTM.

if you squash the commits then i will merge.

**Comment by smiklosovic** [comment]
@michaelsembwever squashed

**Pull request merged**